src/compiler/sexp.rs

```diff
@@ -462,6 +462,14 @@ impl SExp {
         }
     }
 
+    pub fn atomize(&self) -> SExp {
+        match self {
+            SExp::Integer(l, i) => SExp::Atom(l.clone(), u8_from_number(i.clone())),
+            SExp::QuotedString(l, _, a) => SExp::Atom(l.clone(), a.clone()),
+            _ => self.clone(),
```

Comment:
I would think it would be an error to call this on something that isn't atom-like, such as a pair, no?

Reply:
So here's why this is here.
Worst case, atoms are round tripped through clvmr's runner and may be converted to integers or other syntactic variants of atom from the perspective of this compiler, and since that's just executable code the compiler doesn't have full control.  Eventually, src/compiler/evaluate's runner (which also implements the repl) will supercede the clvm runner for running macros and we can be much more strict.  For now, we must allow certain things through to allow use of macros funneled through the clvm value space even though it seems like it wouldn't be needed.